akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/LatencySpec.scala

```diff
@@ -4,8 +4,9 @@
 package akka.remote.artery
 
 import java.util.concurrent.Executors
-import java.util.concurrent.atomic.AtomicLongArray
+import java.util.concurrent.atomic.{ AtomicBoolean, AtomicLongArray }
```

Comment:
Can I just leave it for this PR? I don't want to wait for another PR validation :( (unless you find other issues of course)